content/chainguard/chainguard-images/images-testing.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "How Chainguard Images are Tested"
+linktitle: "How Images are Tested"
+type: "article"
+description: "A conceptual article outlining testing requirements for Chainguard Images."
+date: 2024-03-13T11:07:52+02:00
+lastmod: 2023-03-13T11:07:52+02:00
+draft: false
+tags: ["Overview", "Chainguard Images", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 550
+toc: true
+---
+
+Chainguard Images are minimal, distroless container images that you can use to build and run secure applications. A frequently-asked question about these Images relates to what kind of testing Chainguard performs on them to ensure they match the functionality of their upstream counterparts.
```

Comment:
I think we can make more of a claim here rather than this being an FAQ

```suggestion
Chainguard Images are minimal, distroless container images that you can use to build and run secure applications. Given the importance of secure, highly performant images, Chainguard performs testing to ensure our Images match the functionality of upstream and other external counterparts.
```